tests/template_backends/test_django.py

```diff
@@ -145,3 +146,32 @@ def test_non_debug_default_template_loaders(self):
     def test_debug_default_template_loaders(self):
         engine = DjangoTemplates({'DIRS': [], 'APP_DIRS': True, 'NAME': 'django', 'OPTIONS': {}})
         self.assertEqual(engine.engine.loaders, self.default_loaders)
+
+    @override_settings(DEBUG=True)
+    def test_debug_template_loaders_with_dirs(self):
+        engine = DjangoTemplates({
+            'DIRS': ['dirs'],
+            'APP_DIRS': True,
+            'NAME': 'django',
+            'OPTIONS': {},
+            'POST_APP_DIRS': ['post_app_dirs'],
+        })
```

Comment:
Is there a reason not to suggest using

```python
TEMPLATES = [{
    'NAME': 'django',
    'OPTIONS': {
        'loaders': [
            ('django.template.loaders.filesystem.Loader', ['dir']),
            'django.template.loaders.app_directories.Loader',
            ('django.template.loaders.filesystem.Loader', ['post_app_dirs']),
        ]
    }
}]
```

In the rare cases where a post app-directory loader is required?